fastapi/__init__.py

```diff
@@ -1,7 +1,6 @@
 """FastAPI framework, high performance, easy to learn, fast to code, ready for production"""
 
 __version__ = "0.58.1"
-
```

Comment:
isort is removing this line? If yes, could you check why?

Reply:
I'm not sure but looking into it.

Reply:
This is being fixed by isort team. see https://github.com/timothycrosley/isort/issues/1283